packages/mds-agency-cache/index.ts

```diff
@@ -139,22 +94,20 @@ async function hread(suffix: string, device_id: UUID): Promise<CachedItem> {
 
 /* Store latest known lat/lng for a given device in a redis geo-spatial analysis compatible manner. */
 async function addGeospatialHash(device_id: UUID, coordinates: [number, number]) {
-  const client = await getClient()
   const [lat, lng] = coordinates
-  const res = await client.geoadd(decorateKey('locations'), lng, lat, device_id)
+  const res = await client.geoadd(device_id, lng, lat)
   return res
 }
 
 async function getEventsInBBox(bbox: BoundingBox) {
   const start = now()
-  const client = await getClient()
   const [pt1, pt2] = bbox
   const points = bbox.map(pt => {
     return { lat: pt[0], lng: pt[1] }
   })
   const [lng, lat] = [(pt1[0] + pt2[0]) / 2, (pt1[1] + pt2[1]) / 2]
   const radius = routeDistance(points)
-  const events = client.georadiusAsync(decorateKey('locations'), lng, lat, radius, 'm')
+  const events = client.georadius(decorateKey('locations'), lng, lat, radius, 'm')
```

Comment:
Should this be `await`-ed? `events` is just going to be a promise here, I believe, so it won't be included in the `timeElapsed`.

Reply:
Yep, this looks like it was a mistake from Way Back.  Probably should have types on e.g. `events`.